releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -4088,8 +4517,8 @@
   },
   "125218": {
     "commit": "ba28796e778731bdb8dfeaa9a85454ff28306c1a",
-    "text": "DRA: enhance validation for the ResourceClaimParametersReference and ResourceClassParametersReference with the following rules:\n\n1. `apiGroup`: If set, it must be a valid DNS subdomain (e.g. 'example.com').\n2. `kind` and `name`: It must be valid path segment name. It may not be '.' or '..' and it may not contain '/' and '%' characters.",
-    "markdown": "DRA: enhance validation for the ResourceClaimParametersReference and ResourceClassParametersReference with the following rules:\n  \n  1. `apiGroup`: If set, it must be a valid DNS subdomain (e.g. 'example.com').\n  2. `kind` and `name`: It must be valid path segment name. It may not be '.' or '..' and it may not contain '/' and '%' characters. ([#125218](https://github.com/kubernetes/kubernetes/pull/125218), [@carlory](https://github.com/carlory)) [SIG Node]",
+    "text": "Dynamic Resource Allocator (DRA): Enhanced validation for the ResourceClaimParametersReference and ResourceClassParametersReference with the following rules:\n\n1. `apiGroup`: If set, it must be a valid DNS subdomain (e.g. 'example.com').\n2. `kind` and `name`: It must be valid path segment name. It may not be '.' or '..' and it may not contain '/' and '%' characters.",
+    "markdown": "Dynamic Resource Allocator (DRA): Enhanced validation for the ResourceClaimParametersReference and ResourceClassParametersReference with the following rules:\n  \n  1. `apiGroup`: If set, it must be a valid DNS subdomain (e.g. 'example.com').\n  2. `kind` and `name`: It must be valid path segment name. It may not be '.' or '..' and it may not contain '/' and '%' characters. ([#125218](https://github.com/kubernetes/kubernetes/pull/125218), [@carlory](https://github.com/carlory))",
```

Comment:
```suggestion
    "markdown": "Dynamic Resource Allocator (DRA): Enhanced validation for the ResourceClaimParametersReference and ResourceClassParametersReference with the following rules:\n  \n  1. `apiGroup`: If set, it must be a valid DNS subdomain (e.g. 'example.com').\n  2. `kind` and `name`: It must be valid path segment name. It may not be '.' or '..' and it may not contain '/' and '%' characters. ([#125218](https://github.com/kubernetes/kubernetes/pull/125218), [@carlory](https://github.com/carlory)) [SIG Node]",
```